app/models/ability.rb

```diff
@@ -41,7 +41,7 @@ def grant_permissions
     can %i[manage_item manage_desc_metadata manage_governing_apo view_content view_metadata], [NilModel, Cocina::Models::DRO] if current_user.manager?
     can :create, Cocina::Models::AdminPolicy if current_user.manager?
 
-    can %i[view_metadata view_content], [ActiveFedora::Base, Cocina::Models::DRO] if current_user.viewer?
+    can %i[view_metadata view_content], [ActiveFedora::Base, Cocina::Models::DRO, Cocina::Models::Collection] if current_user.viewer?
```

Comment:
i'm a little rusty on argo permission checking stuff:  i think this makes sense, but why no need to also add `Cocina::Models::AdminPolicy`?

Reply:
Sure, I've added that too.  This only affects people who have the `sdr:viewer-role` and I'm not sure who that is.